edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/config.json

```diff
@@ -24,6 +24,7 @@
         "type": "docker", // Enum - values – docker
         "status": "running", // Enum - values - running, stopped
         "restartPolicy": "always", // This value should be "always"
+        "pullPolicy": "always",
```

Comment:
Let the default be the pullPolicy is not specified

Reply:
sure